### PR TITLE
[sonic-cfggen]: Fix minigraph DhcpRelays parsing issue

### PR DESCRIPTION
**Pull request by pavel-shirshov**
**- What I did**
Add additional check to fix https://github.com/Azure/sonic-buildimage/issues/1302

**- How I did it**
I check .text attribute is not None before using it.

**- How to verify it**
Use sonic-cfggen with minigraph, where DhcpRelays are empty:
`<DhcpRelays></DhcpRelays>`

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by jleveque** [comment]
Retest this please.

**Comment by lguohan** [comment]
@pavel-shirshov , can you fix the sonic-cfggen test?

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -211,7 +211,8 @@ def parse_dpg(dpg, hname):
 
             # If this VLAN requires a DHCP relay agent, it will contain a <DhcpRelays> element
             # containing a list of DHCP server IPs
-            if vintf.find(str(QName(ns, "DhcpRelays"))) is not None:
+            if vintf.find(str(QName(ns, "DhcpRelays"))) is not None and
```
Comment (jleveque) [inline src/sonic-config-engine/minigraph.py]: Line break in the middle of `if` statement requires parentheses around entire statement or backslash to escape newline. This is the cause of the check build failures.

Comment (pavel-shirshov) [reply src/sonic-config-engine/minigraph.py]: you're right. Refactored

**Pull request merged**